.github/actions/local-aggregator-deploy/action.yml

```diff
@@ -4,14 +4,10 @@ description: Runs an aggregator instance
 runs:
   using: composite
   steps:
-    - working-directory: ./
-      shell: bash
-      run: docker-compose up -d postgres
-
     - working-directory: ./aggregator
       shell: bash
       run: cp .env.test .env
 
     - working-directory: ./aggregator
       shell: bash
-      run: deno run --allow-read --allow-env --allow-net ./programs/aggregator.ts 2>&1 | tee -a aggregatorLogs.txt &
+      run: deno run --allow-read --allow-write --allow-env --allow-net ./programs/aggregator.ts 2>&1 | tee -a aggregatorLogs.txt &
```

Comment:
There are likely additional deno scripts in `./aggregator/manualTests` & `./aggregator/programs` that need `--allow-write` added to their bash run command (`#!/usr/bin/env -S deno...`) if they interact with the db.

Reply:
Nice catch.

Includes `aggregator.ts` 😅.

Noticed and fixed a few other things: https://github.com/web3well/bls-wallet/pull/516.